Makefile

```diff
@@ -8,12 +8,9 @@ clean:
 	scons --clean
 	rm -rfv ./dist ./statsite ./rpm-build ./.sconsign.dblite ./.sconf_temp
 
-test_runner:
+test:
 	scons test_runner
 
-test: test_runner
-	./test_runner
```

Comment:
I think this line still needs to be present, otherwise we are compiling but not running the tests